.github/workflows/main.yml

```diff
@@ -11,7 +12,6 @@ jobs:
       GDRIVE_FOLDER: ${{ vars.GDRIVE_FOLDER }}
     steps:
     - uses: actions/checkout@v3
-    - run: 'echo "hello"'
     - run: "pip install -r gdrive_requirements.txt"
```

Comment:
No quotes needed here. Sorry, I realize this is not specifically in the scope of these changes, but I noticed it.

Reply:
removed quotes